Tests fail on node 5

```
$ nvm version
v5.12.0
```

```
> postcss-prefix@2.1.0 test /Users/cjhowe7/Desktop/postcss-prefix
> tape test/index.js

TAP version 13
# postcss-prefix
/Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss/lib/lazy-result.js:277
            throw error;
            ^

SyntaxError: Identifier 'prefixNode' has already been declared
    at /Users/cjhowe7/Desktop/postcss-prefix/index.js:19:35
    at /Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss-selector-parser/dist/selectors/container.js:164:26
    at Root.each (/Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss-selector-parser/dist/selectors/container.js:147:22)
    at Root.eachInside (/Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss-selector-parser/dist/selectors/container.js:163:21)
    at Processor.transformSelectors [as func] (/Users/cjhowe7/Desktop/postcss-prefix/index.js:19:15)
    at Processor.process (/Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss-selector-parser/dist/processor.js:31:14)
    at /Users/cjhowe7/Desktop/postcss-prefix/index.js:12:10
    at /Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss/lib/container.js:241:28
    at /Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss/lib/container.js:148:26
    at Root.each (/Users/cjhowe7/Desktop/postcss-prefix/node_modules/postcss/lib/container.js:114:22)
npm ERR! Test failed.  See above for more details.
```

--- a/index.js
+++ b/index.js
@@ -1,3 +1,5 @@
+'use strict';
+
 const postcss = require('postcss')
 const Selector = require('postcss-selector-parser')
 

